docs/cip/README.md

```diff
@@ -247,6 +247,91 @@ As a result of this decoupled approach, Leios can utilize nearly the full bandwi
+
+$$`
+p_\text{noquorum} = \text{probability of sufficient voters to achieve a quorum in a given pipeline}
+`$$
+
+***Network egress:*** Cloud service providers typically charge for network egress rather than for network ingress. The quantity $`q_\text{egress}`$ is simply the number of bytes sent from a node per unit time.
+
+***Disk usage:*** Leios requires that IBs, EBs, and RBs be stored permanently; votes need not be stored permanently, however. The quantity $`q_\text{disk}`$ is the total number of IB, EB, and RB bytes generated per unit time.
+
+***I/O operations:*** Some cloud service providers limit or bill input/output operations on a per-second capacity basis. The number of I/O operations depends upon the filesystem's block size $`b`$, not on the logical number of writes to disk by the protocol: e.g., writing an IB of 32,768 bytes might consist of 64 I/O operations on a filesystem having a 512-byte block size. We assume that disk caching and delayed writes smooth out the unevenness in I/O operations, so that the mean $`\bar{q}_\text{iops}`$ is the best metric here.
+
+***Mean CPU usage:*** Computation resources consumed by the number are quantified as $`\bar{q}_\text{vcpu}`$, which is the mean number of virtual CPU cores utilized by the protocol.
+
+***Peak CPU usage:*** Because CPU usage varies depending upon the node's activity, the maximum number of virtual CPU cores utilized by the protocol during any slot, $`\hat{q}_\text{vcpu}`$, provides a useful indication of computational burstiness and of how a virtual machine should be sized for Leios.
+
+***Bandwidth:*** If the bandwidth for inter-node communication drops below a given value, then the throughput of Leios (at a given level of demand) will be drop, as network congesting occurs.
+
+$$`
+\eta_\text{bandwidth}(b) = \frac{\text{bytes of transactions reaching the ledger if links have bandwidth } b}{\text{bytes of transactions reaching the ledger if bandwidth were infinite}}
+`$$
+
+***Conflict:*** Theoretically, an adversary could poison the global memory pool by simultaneously submitting mutually conflicting transactions to every node in the network. This would result in massive conflicts in the contents of IBs produced during the concurrency window, resulting in much wasted space in IBs, effectively reducing the throughput of the protocol.
+
+$$`
+\hat{\eta}_\text{conflict} = \frac{\text{bytes of valid transactions reaching the ledger}}{\text{bytes of transactions reaching the ledger if no conflicts were present}}
+`$$
+
+***Message loss:*** Using the same metric $`\eta`$, one can also measure the loss in throughput due to unreliable network infrastructure where the fraction $`\ell`$ of messages sent from a node are lost before reaching their destination node.
+
+$$`
+\eta_\text{message}(\ell) = \frac{\text{bytes of transactions reaching the ledger if } \ell \text{ fraction of messages are lost}}{\text{bytes of transactions reaching the ledger if no messages are lost}}
+`$$
```

Comment:
Why should we need this? There is no message loss in our network protocols (as all of them are pull-based and a node would re-request incompletely downloaded data).